File: Task2.rb
def count_digit(num)
  digit_len = num.abs.to_s.length
  digit_len % 2 != 0 ? 1 : 0
end

print "Enter the numbers with a space: "
entry_num = gets.chomp
nums = entry_num.split.map(&:to_i)
count_odd_digits = nums.map { |num| count_digit(num) }.sum
puts count_odd_digits

File: Task2.py
def Count_Digit(Num):
    digit_len = len(str(abs(Num)))
    if digit_len % 2 != 0:
        return 1
    else:
        return 0

entry_num = input("Enter the numbers with a space: ")
Nums = [int(num) for num in entry_num.split()]

count_odd_digits = sum(Count_Digit(num) for num in Nums)
print(count_odd_digits)
